sdk/src/androidTest/java/com/karhoo/sdk/api/testrunner/TestSDKConfig.kt

```diff
@@ -25,4 +26,8 @@ class TestSDKConfig(val context: Context, private val authenticationMethod: Auth
     override fun analyticsProvider(): AnalyticProvider? {
         return null
     }
+
+    override suspend fun requireSDKAuthentication(callback: () -> Unit) {
+        //Do nothing
```

Comment:
@AleksanderWedrychowskiKarhoo Looks like there's nothing to do where my TODO was added. Can you please remove it from iOS? Or if you have any ideas already let us know and we'll update Android